test/test.ebuild

```diff
@@ -0,0 +1,200 @@
+# Copyright 2021 Gentoo Authors
```

Comment:
I don't think this file is being actually used anywhere, is it?

Reply:
I believe that was added automatically when I ran `vim test.ebuild` to create the file.

Reply:
I meant the whole file.

Reply:
OK, I've removed it.